### PR TITLE
Use response event in configuration

### LINKED ISSUE #168: getting-started no longer correct?
The breaking change in good which should have been fixed by pull request "Update good-console example #164" only prevents start-up error. Functionality of the getting-started (logging) tutorial appears to still be broken.

#### Symptoms
Tutorial says
````
And if we visit http://localhost:3000/ in the browser, you'll see:

140625/143205.774, request, http://localhost:3000: get / {} 200 (10ms)
````

but this log never appears.

#### Initial analysis
1. hapi seems to now propagate "request" events as "request-internal"
2. good appears not to propagate "request-internal" to reporters. I may misunderstand the code...

#### Context
Windows 7
node 0.12
hapi 8.4.0
good 6.1.0
good-console 5.0.0

### PR DESCRIPTION
**Pull request by gergoerdosi**
Closes #168.

### CODE REVIEW

File: lib/tutorials/getting-started.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ Now when the server is started you'll see:
 And if we visit `http://localhost:3000/` in the browser, you'll see:
 
 ```
-140625/143205.774, request, http://localhost:3000: get / {} 200 (10ms)
+140625/143205.774, response, http://localhost:3000: get / {} 200 (10ms)
```
Comment (goofballLogic) [inline lib/tutorials/getting-started.md]: Note I also see `[response]` rather than `response`

Comment (AdriVanHoudt) [inline lib/tutorials/getting-started.md]: ^correct also no `:` after the url

**Comment by gergoerdosi** [comment]
@goofballLogic, @AdriVanHoudt: Console output is character perfect now. Thanks.

**Comment by AdriVanHoudt** [comment]
:+1:

**Pull request merged**